appveyor.yml

```diff
@@ -88,6 +88,8 @@ install:
     - cmd: conda install --yes --quiet obvious-ci conda-build-all
     - cmd: conda install --yes --quiet conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
+    # current workaround for issue #1130 in conda-build-all
+    - cmd: conda install --yes --quiet conda-build=1.21.5
```

Comment:
If it works here, we should close PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/19 ) as it is not needed either.